conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,3816 @@
+    io:println(s1 << i2); // @output 32640
+    io:println(i2 << s2); // @output 28
+    io:println(s2 << b1); // @output 8
+    io:println(b1 << i2); // @output 256
+
+    io:println((i3 << s1).toBalString()); // @output ()
+    io:println((s2 << s3).toBalString()); // @output ()
+    io:println((b2 << i2).toBalString()); // @output ()
+    io:println((s3 << i3).toBalString()); // @output ()
+    io:println((i3 << b2).toBalString()); // @output ()
+}
+
+Test-Case: output
+Description: Test the '<<' expression at module level.
+Labels: byte, int, int:Signed16, int:Signed32, int:Signed8, int:Unsigned16, int:Unsigned32,
+        int:Unsigned8, shift-expr, unary-minus
+
+int i1 = 1221112;
+int i2 = 3;
+
+int:Signed8 s1 = 123;
+int:Unsigned8 s2 = 5;
+int:Signed16 s3 = -8;
+int:Unsigned16 s4 = 3;
+int:Signed32 s5 = -9;
+int:Unsigned32 s6 = 17;
+
+byte b1 = 255;
+byte b2 = 4;
+
+int r1 = i1 << i2;
+
+int r2 = s1 << s2;
+int r3 = s3 << s4;
+int r4 = s5 << s6;
+int r5 = i1 << s1;
+int r6 = s6 << i2;
+
+int r7 = b1 << b2;
+int r8 = b1 << i2;
+int r9 = i1 << b2;
+int r10 = s1 << b2;
+int r11 = b1 << s4;
+
+int r12 = 12331321 << 3;
+
+function init() {
+    io:println(r1); // @output 9768896
+    io:println(r2); // @output 3936
+    io:println(r3); // @output -64
+    io:println(r4); // @output -1179648
+    io:println(r5); // @output -4611686018427387904
+    io:println(r6); // @output 136
+    io:println(r7); // @output 4080
+    io:println(r8); // @output 2040
+    io:println(r9); // @output 19537792
+    io:println(r10); // @output 1968
+    io:println(r11); // @output 2040
+    io:println(r12); // @output 98650568
+}
+
+Test-Case: output
+Description: Test '<<' expression with field access as operands.
+Labels: DecimalNumber, int, mapping-constructor-expr, field-access-expr,
+        module-type-defn, record-type, shift-expr
+
+function init() {
+    Record a = {x: 10};
+
+    int c = a.x << 3;
+    io:println(c); // @output 80
+    int d = 40 << a.x;
+    io:println(d); // @output 40960
+}
+
+type Record record {
+    int x;
+};
+
+Test-Case: output
+Description: Test '<<' expression with member access as operands.
+Labels: array-type, DecimalNumber, int, list-constructor, shift-expr, member-access-expr
+
+function init() {
+    int[] b = [3, 40];
+
+    int c = 10 << b[0];
+    io:println(c); // @output 80
+    int d = b[1] << 10;
+    io:println(d); // @output 40960
+}
+
+Test-Case: output
+Description: Test '<<' expression with function call as operands.
+Labels: DecimalNumber, function-call-expr, function-defn, int, shift-expr
+
+function init() {
+    int b = 10 << getInt();
+    io:println(b); // @output 40
+    int c = getInt() << 10;
+    io:println(c); // @output 2048
+}
+
+function getInt() returns int {
+    return 2;
+}
+
+Test-Case: output
+Description: Test '<<' expression with method call as operands.
+Labels: DecimalNumber, explicit-new-expr, int, method-call-expr, module-class-defn, shift-expr
```

Comment:
Need `return-stmt` label.

Reply:
Since it is in minimal subset, we can skip the label.